imgui/build.gradle.kts

```diff
@@ -1,70 +1,76 @@
-import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
-import org.jetbrains.kotlin.konan.target.HostManager
+import org.jetbrains.kotlin.gradle.plugin.mpp.*
+import org.jetbrains.kotlin.konan.target.*
 
 plugins {
-    kotlin("multiplatform")
-    `maven-publish`
+	kotlin("multiplatform")
+	`maven-publish`
 }
 
 val useSingleTarget: Boolean by rootProject.extra
 
 kotlin {
-    if (!useSingleTarget || HostManager.hostIsLinux) linuxX64()
-    if (!useSingleTarget || HostManager.hostIsMingw) mingwX64()
-    if (!useSingleTarget || HostManager.hostIsMac) macosX64()
+	if (!useSingleTarget || HostManager.hostIsLinux) linuxX64()
+	if (!useSingleTarget || HostManager.hostIsMingw) mingwX64()
+	if (!useSingleTarget || HostManager.hostIsMac) macosX64()
 
-    jvm {
```

Comment:
I like tabs too (although GitHub doesn't) but let's do this in a separate PR to master.

Reply:
It might be worth adding an editorconfig file to the project as well. I use auto-format all the time, so I've had a bunch of places where formatting was changed because I use the kotlin style guide options which aren't the default in IntelliJ

Reply:
Can commit the code style settings in the `.idea` folder.

Reply:
That's definitely an option, but out of experience it's easier and more transparent to configure the settings in .editorconfig files, and *all* of the Intellij code style settings can be set from there.

For example, these are the settings I use for my projects:

```editorconfig
root = true

[*]
charset = utf-8

# tabs for indentation, spaces for alignment
# set tab width to whatever you want
indent_size = tab
indent_style = tab
ij_smart_tabs = true

insert_final_newline = true
max_line_length = 120

# formatter control
ij_formatter_off_tag = @formatter:off
ij_formatter_on_tag = @formatter:on
ij_formatter_tags_enabled = true

# use official kotlin style
ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL

# use wildcard imports exclusively
# if imports get collapsed by the IDE anyway, there's no reason to individually import anything
ij_kotlin_name_count_to_use_star_import = 1
ij_kotlin_name_count_to_use_star_import_for_members = 1
```

Putting that in a file `.editorconfig` in the project root directory will make sure all contributors have consistent style settings for the project without being tied to the `.idea` folder that may get deleted in an effort to reimport Gradle (as I've only recently had to do)

Reply:
I don't know what style settings you prefer, otherwise I'd go ahead and make a PR.

Reply:
Yeah those settings are fine.